javaScript/signup.js

```diff
@@ -0,0 +1,134 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const emailInput = document.getElementById("email");
+  const emailError = document.getElementById("email-error");
+
+  const usernameInput = document.getElementById("username");
+  const usernameError = document.getElementById("username-error");
+
+  const passwordInput = document.getElementById("password");
+  const passwordError = document.getElementById("password-error");
+
+  const confirmPasswordInput = document.getElementById("confirm-password");
+  const confirmPasswordError = document.getElementById("confirm-password-error");
+
+  const passwordToggle = document.querySelector("#password-toggle");
```

Comment:
💊 제안
해당 변수가 태그임을 일 수 있도록 명명해주시면 더 좋을 것 같아요~

```suggestion
  const passwordToggleButton = document.querySelector("#password-toggle");
```